src/internal/components/masked-input/utils/mask-format.ts

```diff
@@ -60,33 +62,33 @@ class MaskFormat {
     return inputSegments.every((segmentValue, i) => {
       const segment = this.segments[i];
 
-      // disallow empty segments
+      // Disallow empty segments
```

Comment:
suggest removing these writing changes, I'm not sure they serve any real purpose and just end up making the diff/history more difficult to follow